### PR TITLE
Changed phone number route names

### PR DESCRIPTION
**Pull request by AdamP-HRSDC-RHDCC**
- Changed phone number route names to better conform with other personal info pages
- Removed PhoneNumber component
- Added html definition list for readonly text
- Fixed associated unit tests

### CODE REVIEW

File: frontend/__tests__/actions/edit-phone-number.test.tsx [hunk 1]
```diff
@@ -1,40 +1,33 @@
...
-
-describe('Update Information Action', () => {
+describe('Edit Phone Number Action', () => {
   afterEach(() => {
     vi.clearAllMocks();
     vi.resetModules();
   });
 
-  it('Should redirect if no validation errors', async () => {
+  it('Should redirect without validation erros', async () => {
```
Comment (gregory-j-baker) [inline frontend/__tests__/actions/edit-phone-number.test.tsx:13]:
```suggestion
  it('Should redirect without validation errors', async () => {
```

**Review by gregory-j-baker — APPROVED** [review]
Typo in one of your test names. Sorry.

File: frontend/__tests__/actions/edit-phone-number.test.tsx (hunk 1 above)
Comment (gregory-j-baker) [reply frontend/__tests__/actions/edit-phone-number.test.tsx:13]: Just self-resolve when you fix it.. I don't need to re-review.

Comment (AdamP-HRSDC-RHDCC) [reply frontend/__tests__/actions/edit-phone-number.test.tsx]: Fixed the typo

**Review by gregory-j-baker — APPROVED** [review]

**Pull request merged**